### PR TITLE
Fix server list and use common NGINX command

### PR DESCRIPTION
**Pull request by MSevey**
- `eu-ger-8.siasky.net` had a typo
- I split out the siasky.dev servers 
- I copied over the NGINX command from the airtable command

### CODE REVIEW

File: scripts/blocklist-skylink.sh [hunk 1]
```diff
@@ -32,21 +37,23 @@ fi
...
-                      "siasky.xyz" "siasky.dev")
+                      "siasky.xyz" "dev1.siasky.dev" "dev2.siasky.dev" "dev3.siasky.dev")
 for server in "${servers[@]}";
 do
     for skylink in "${skylinks[@]}";
     do
         echo ".. ⌁ Blocking skylink ${skylink} on ${server}"
-
-        ssh -q -t user@${server} "docker exec sia siac skynet blocklist add $skylink && docker exec nginx curl -s -i -X PURGE http://localhost/$skylink | egrep \"^(OK|HTTP|X-)\""
+        cached_files_command="find /data/nginx/cache/ -type f | xargs --no-run-if-empty grep -Elsq '^Skynet-Skylink: ${skylink}'"
```
Comment (kwypchlo) [inline scripts/blocklist-skylink.sh:53]: is `q` searching just for first file and it stops or it just searches files until it finds a match but still searches all files?

Comment (kwypchlo) [reply scripts/blocklist-skylink.sh:53]:
```suggestion
        cached_files_command="find /data/nginx/cache/ -type f | xargs -r grep -Elsq '^Skynet-Skylink: ${skylink}'"
```

shorter syntax

Comment (MSevey) [reply scripts/blocklist-skylink.sh:53]: > is `q` searching just for first file and it stops or it just searches files until it finds a match but still searches all files?

q was to suppress the output. Running it manually I didn't want to see all the files that were being grepped.

Comment (MSevey) [reply scripts/blocklist-skylink.sh]: > shorter syntax

added

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**